code/modules/urist/modules/newtrading/trade/trade_items/artifact dealer.dm

```diff
@@ -0,0 +1,154 @@
+/datum/trade_item/artifacts
+	quantity = 0
+	trader_weight = 1
+	value = 500
```

Comment:
I'm personally not fond of this approach for readability. I'd rather the value is defined on each type itself, than some types inheriting from here and others not

Reply:
removing values where they default to worths_list value